tests/model_fields/tests.py

```diff
@@ -411,13 +411,6 @@ def test_empty_iterator_choices(self):
         self.assertEqual(WhizIterEmpty(c=None).c, None)    # Blank value
         self.assertEqual(WhizIterEmpty(c='').c, '')        # Empty value
 
-    def test_charfield_get_choices_with_blank_iterator(self):
```

Comment:
i believe this test was only accidentally working, due to "empty" generators being truthy. E.g. calling `get_choices` with `choices` provided as an empty list raises an exception before this patch, and now "empty" generators have the same behaviour